src/bacnet/config.h

```diff
@@ -102,6 +104,27 @@
 #endif
 #endif
 
+#if defined(BACDL_BSC)
+#ifndef SC_NETPORT_BVLC_MAX
+#define SC_NETPORT_BVLC_MAX 1500
+#endif
+#ifndef SC_NETPORT_NPDU_MAX
+#define SC_NETPORT_NPDU_MAX 1500
+#endif
+#ifndef SC_NETPORT_CONNECT_TIMEOUT
+#define SC_NETPORT_CONNECT_TIMEOUT 5
+#endif
+#ifndef SC_NETPORT_HEARTBEAT_TIMEOUT
+#define SC_NETPORT_HEARTBEAT_TIMEOUT 3
+#endif
```

Comment:
The Heartbeat should be 60 s.
Tested with Loytec (Cimetrics BACstac runtime v6.4.2.6).
If the Heartbeat is greater than 150s (multiply 2 in bsc_process_srv_awaiting_request() 300s) than
crash the Cimetrics BACstac after 300s. Other People Problem.

But in bsc-socket.h is
```
    /* According 12.56.Y10 SC_Heartbeat_Timeout */
    /* (http://www.bacnet.org/Addenda/Add-135-2020cc.pdf) the recommended default */
    /* value is 300 seconds. */
```
The default is 300s. In this case SC_NETPORT_DISCONNECT_TIMEOUT should be 150s.